features/other_a/profile_edit.feature

```diff
@@ -86,6 +86,9 @@ Scenario: Changing email address -- can't be the same as another user's
   When I enter a duplicate email
   Then I should see "Email has already been taken"
     And 0 emails should be delivered
+    # Regression test for AO3-5049
+    And I should not see "Email addresses don't match!"
+    And I should not see "foo@ao3.org"
```

Comment:
Can you add steps to check that the correct error message ("Email has already been taken") and the current email (`bar@ao3.org`) show up?

Reply:
> "Email has already been taken"
That check is already there on line 87.

> and the current email (bar@ao3.org) show up
I've added this one.